### PR TITLE
feat: added get layer base logic + tests

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                       |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✔                                                                        |
| Chore            | ✔                                                                       |

### CODE REVIEW

File: src/layers/controllers/layersController.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ export class LayersController {
   public constructor(@inject(Services.LOGGER) private readonly logger: ILogger, @inject(LayersManager) private readonly manager: LayersManager) {}
 
   public getLayer: GetLayerHandler = (req, res) => {
-    return res.status(httpStatus.OK).json(this.manager.getLayer());
+    return res.status(httpStatus.OK).json(this.manager.getLayer(req.params.name));
```
Comment (asafMasa) [inline src/layers/controllers/layersController.ts:19]: add try catch

File: src/layers/models/layersManager.ts [hunk 2]
```diff
@@ -24,9 +23,14 @@ export class LayersManager {
     @inject(Services.MAPPROXY) private readonly mapproxyConfig: IMapProxyConfig
   ) {}
 
-  public getLayer(): ILayerPostRequest {
-    this.logger.log('info', 'Get layer request');
-    return mockLayer;
+  public getLayer(layerName: string): IMapProxyCache {
+    this.logger.log('info', `Get layer '${layerName}' request`);
```
Comment (asafMasa) [inline src/layers/models/layersManager.ts:27]: remove info message

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: src/layers/models/layersManager.ts (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply src/layers/models/layersManager.ts]: fixed in ae8b206

File: src/layers/controllers/layersController.ts (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply src/layers/controllers/layersController.ts]: fixed in 94c8532

**Review by asafMasa — APPROVED** [review]

**Pull request merged**